pkg/commands/operation/operation.go

```diff
@@ -21,22 +21,22 @@ import (
 var mu sync.Mutex
 
 // DownloadDB downloads the DB
-func DownloadDB(ctx context.Context, appVersion, cacheDir string, dbRepository name.Reference, quiet, skipUpdate bool,
+func DownloadDB(ctx context.Context, appVersion, cacheDir string, dbRepositories []name.Reference, quiet, skipUpdate bool,
 	opt ftypes.RegistryOptions) error {
 	mu.Lock()
 	defer mu.Unlock()
 
 	ctx = log.WithContextPrefix(ctx, "db")
 	dbDir := db.Dir(cacheDir)
-	client := db.NewClient(dbDir, quiet, db.WithDBRepository(dbRepository))
+	client := db.NewClient(dbDir, quiet, db.WithDBRepository(dbRepositories))
 	needsUpdate, err := client.NeedsUpdate(ctx, appVersion, skipUpdate)
 	if err != nil {
 		return xerrors.Errorf("database error: %w", err)
 	}
 
 	if needsUpdate {
 		log.InfoContext(ctx, "Need to update DB")
-		log.InfoContext(ctx, "Downloading DB...", log.String("repository", dbRepository.String()))
+		log.InfoContext(ctx, "Downloading DB...", log.Any("repositories", dbRepositories))
```

Comment:
Looks like we can delete this log since we will show it later:
https://github.com/aquasecurity/trivy/blob/95df470087ce12b6554403058329b4c89082caab/pkg/db/db.go#L236

Reply:
Done https://github.com/aquasecurity/trivy/pull/7605/commits/0c5ad869df3ef6b618783e9ed19df1f1894b7a16